modules/odo-new-commands-30.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-new-commands-30_{context}"]
+= New commands in odo 3.0
```

Comment:
I love the way the section is structured.
A suggestion: Can we change the title to make it feel like it belongs to the Understanding section? I find the emphasis on commands in the title a bit misleading. Just reading the title I thought it belonged to the CLI reference section. Also the section highlights the key features too though they are part of those commands.
How about Major highlights of odo 3.0?